bpaotu/bpaotu/views.py

```diff
@@ -243,6 +247,97 @@ def parse_float(s):
         taxonomy_filter=taxonomy_filter), errors)
 
 
+def param_to_filters_without_checks(query_str):
+    def parse_date(s):
+        try:
+            return datetime.datetime.strptime(s, '%Y-%m-%d').date()
+        except ValueError:
+            return datetime.datetime.strptime(s, '%d/%m/%Y').date()
+
+    def parse_float(s):
+        try:
+            return float(s)
+        except ValueError:
+            return None
+
```

Comment:
are these copied out? I think we should move them into a shared file and then import them if so

Reply:
Fixed.